src/components/MarketCapCard.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Card } from 'semantic-ui-react';
+import MarketChart from './MarketChart'
+import { getMarketCapData } from '../modules/datacenter'
+
+class MarketCapChart extends React.Component {
+  state = {
+    market_data: []
+  }
+  componentDidMount = async () => {
+    try {
+      let response = await getMarketCapData()
+      this.setState({ market_data: response.market_data })
+    }
+    catch (error) {
+      this.setState({ errorMessage: error.message })
+    }
+  }
+
+  render() {
+    const { market_data, errorMessage } = this.state
+    let daily_cap = market_data[1] ? (+market_data[market_data.length - 1].market_cap).toLocaleString() : errorMessage
```

Comment:
Can you explain the `+` sign? What does it do?

Reply:
It's shorthand that parses string into number so that we can then convert back to a string with thousand seperators with toLocaleString()